recipes/rsgislib/bld.bat

```diff
@@ -0,0 +1,45 @@
+
+# attempt to debug cmake problem
+echo %PATH%
+where cmake
```

Comment:
:tada: 

`proj.4` was probably one of the oldest recipes we had from before conda-forge even existed. At that time we relied on AppVeyor's `cmake` and did not use `curl` in the recipe, so that copy was harmless. Good that we caught and fixed that! Thanks @gillins and @jschueller!